haskell-open-source.md

```diff
@@ -0,0 +1,35 @@
+## Best Practices
+
+1. If authoring a distinct package of reasonable size and generic utility,
+   prefer starting a separate, public repository utilized as a Git dependency
```

Comment:
The line here seems fuzzy. I feel like things often start out as small utilities and grow into something that should be made into a library/module. Where that happens I have no idea.

Reply:
I'm happy to change it (or remove it). I'm trying to put just a little bit of pressure to move away from the failure mode of starting with a sub-directory and never extracting it. Maybe we don't care about that risk?

Reply:
:man_shrugging: